src/platform-implementation-js/views/compose-view.ts

```diff
@@ -225,11 +275,11 @@ export default class ComposeView extends EventEmitter {
     return get(memberMap, this).composeViewImplementation.getThreadID()!;
   }
 
-  getDraftID(): Promise<string | null | void> {
+  getDraftID(): Promise<string | undefined | null> {
```

Comment:
These type changes bring the external typedef usage (`inboxsdk.d.ts`) types into source code.